packages/elements/src/combo-box/__test__/combo-box.value.test.js

```diff
@@ -132,5 +132,19 @@ describe('combo-box/Value', function () {
       el.value = '';
       expect(el.value).to.equal('', 'Value must be empty string when reset value on free text mode');
     });
+    it('Cannot set any value without free text mode when data is empty', async function () {
```

Comment:
It seems this test suite breaks our "it should" convention. Let's gradually fix this be applying the convention to new test cases.
```suggestion
    it('Should not update value when data is empty', async function () {
```